package.json

```diff
@@ -1,45 +1,47 @@
 {
-  "name": "edu-change-detection",
+  "name": "change-detection",
   "version": "0.0.0",
-  "license": "MIT",
   "scripts": {
-    "build-ghpages": "ng build --prod --base-href \"https://danielwiehl.github.io/edu-angular-change-detection/\"",
-    "deploy-ghpages": "ngh --message=\"Deploy to https://danielwiehl.github.io/edu-angular-change-detection/\""
```

Comment:
Is there a reason why you removed GitHub deployment scripts?

Reply:
some files are generated by with angular cli, I have fixed it, feel free to ping me if other codes need to change